com.locomalito.abbayedesmorts.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.locomalito.abbayedesmorts",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
```

Comment:
Any reason for not using 19.08?

Reply:
Hello, this is my first time creating a flatpak. No special reason to use 18.08, the game that I used as a template used this version. I would have to test if it works with 19.08

Reply:
It does work. Updating runtime, thanks.

Reply:
updated in ee96dc1fee4fc891b121e85445d215561ce2075a